content/en/docs/tutorials/configuration/kubelet-standalone.md

```diff
@@ -0,0 +1,399 @@
+---
+title: Running Kubelet in Standalone Mode
+content_type: tutorial
+weight: 5
+card:
+  name: tutorials
+  weight: 10
+---
+
+<!-- overview -->
+
+This tutorial shows you how to run a standalone Kubelet instance.
+
+In standalone mode the Kubelet allows you to manage containers using pod manifests, which brings the benefits of running tightly coupled application as a single unit while leveraging a subset of advanced features including init containers, CNI networking, etc., without the need for the entire Kubernetes feature set.
+
+## {{% heading "objectives" %}}
+
+* Install and configure `containerd`, `cni-plugins`, and `kubelet` on a Linux system.
+* Run `containerd` and `kubelet` as `systemd` services.
+* Launch a Pod running `nginx` that listens to requests on TCP port 80 on the Pod's IP address.
+
+## {{% heading "prerequisites" %}}
+
+- Admin (root) access to a Linux system (virtual, cloud, or physical instance) running `systemd`.
+- Access to the Internet to download the various components needed for the tutorial.
+- Required CLI tools: `curl`, `tar`.
+- Suggested CLI tools: `jq`.
+
+
+<!-- lessoncontent -->
+
+## Download, install, and configure the components
+
+{{< note >}}
+Download the latest available versions of the required packages.
+The relative path to the downloaded software is assumed to be the `sources` directory.
+Run all the commands as the `root` user.
+{{< /note >}}
+
+```shell
+mkdir sources
+cd sources/
+```
+
+### CNI plugins
+
+- Download:
+
+[releases](https://github.com/containernetworking/plugins/releases)
+
+```shell
+curl -LO https://github.com/containernetworking/plugins/releases/download/v1.5.1/cni-plugins-linux-amd64-v1.5.1.tgz
+```
+
```

Comment:
Please indent the list item contents properly. Same below.

````suggestion
- Download:

  [releases](https://github.com/containernetworking/plugins/releases)

  ```shell
  curl -LO https://github.com/containernetworking/plugins/releases/download/v1.5.1/cni-plugins-linux-amd64-v1.5.1.tgz
  ```

````

Reply:
Actually, I realize it was a mistake to use list items to highlight those entries, I rather use plain text, like in other tutorial pages here. Thanks.